frontend/src/data.ts

```diff
@@ -87,20 +87,36 @@ export function getTotalCost(unitCounts: UnitCounts): number {
     );
 }
 
+function sumUnitCountEntries(unitCountEntries: [Unit, number][]): number {
+    return unitCountEntries.reduce((sum, [, count]) => sum + count, 0);
+}
+
 export function getTotalCapacity(unitCounts: UnitCounts): number {
-    return Array.from(unitCounts).reduce((acc, [, b]) => acc + b, 0);
+    return sumUnitCountEntries(Array.from(unitCounts));
+}
+
+export function getFighterCount(unitCounts: UnitCounts): number {
+    return sumUnitCountEntries(
+        Array.from(unitCounts).filter(([unit]) => unit === Unit.FIGHTER)
+    );
 }
 
 export function getFleetSupplyRemaining(
     currentFleetSupply: number,
     maxFleetSupply: number,
-    unitCounts: UnitCounts
+    unitCounts: UnitCounts,
+    airborneFighterCount: number
```

Comment:
That's perfect, I was so stuck on this name and could only think of `nonTransportedFighters` but didn't like the negation. And airborne isn't even correct because they're in space? Lol, fixed now